client/src/components/VerbContainer.jsx

```diff
@@ -62,89 +160,104 @@ export default class VerbContainer extends Component {
     }
   }
 
-  toggleTense(name) {
-    this.setState((prevState) => {
-      const newState = {
-        tenses: {
-          ...prevState.tenses,
-          [name]: !prevState.tenses[name]
-        }
-      }
-      localStorage.setItem(
-        'tenses',
-        JSON.stringify(Object.assign(prevState.tenses, newState.tenses))
-      );
-      return newState;
-    });
+  toggleTense(tenseNames) {
+    if (tenseNames instanceof Array) {
+      // from the mobile select dropdown
+      this.setState((prevState) => {
+        const tenses = {};
+        Object.keys(prevState.tenses).forEach(t => (tenses[t] = false));
+        tenseNames.forEach(t => tenses[t.value] = true);
+        const newState = { tenses };
+         localStorage.setItem(
+           'tenses',
+           JSON.stringify(tenses)
+        );
+         return newState;
+      });
+      
+    } else {
+      this.setState((prevState) => {
+        const newState = {
+          tenses: {
+            ...prevState.tenses,
+            [tenseNames]: !prevState.tenses[tenseNames]
+          }
+         }
+         localStorage.setItem(
+           'tenses',
+           JSON.stringify(Object.assign(prevState.tenses, newState.tenses))
```

Comment:
instead of Object.assign you can also do `{ ...prevState.tenses, ...newState.tenses }`

Reply:
It's the future